en/asset/compress-texture.md

```diff
@@ -1,43 +1,75 @@
-
-The default settings of `macro.SUPPORT_TEXTURE_FORMATS` is only supported by `pvr` on the iOS platform. This means `pvr` images will only be loaded on iOS browsers, and browsers on other platforms will load `png` images.
\ No newline at end of file
+# Compress texture
+
+**Cocos Creator 3D** can set the compression method required for textures directly in the editor, and then automatically compress the textures when the project is released. For the web platform, multiple image formats can be exported at the same time, and the engine will automatically download the appropriate format according to different browsers.
+
+## Configure compressed texture
+
+**Cocos Creator 3D** supports importing images in multiple formats (see the table below for details), but in an actual running game, we do not recommend using the original images as assets to load. For example, on a mobile platform, only 80% or less of the original image quality may be required, or a `.png` without the transparent channel can be converted into a `.jpg`, which can reduce the storage space required.
+
+| Image format | Android | iOS | Mini Game | Web  | Mac & Windows |
+| ------- | -------- | ------ | ----- | ------ | ------ |
+| PNG | Supported | Supported   | Supported | Supported | Supported |
+| JPG | Supported | Supported | Supported | Supported | Supported |
+| WEBP | Native Supported for Android 4.0+<br>Other versions can use [this library](https://github.com/alexey-pelykh/webp-android-backport) | can use [this library](https://github.com/carsonmcdonald/WebP-iOS-example) | Supported | [Partially Supported](https://caniuse.com/#feat=webp) | Not Supported |
+| PVR | Not Supported | Supported | Supported iOS  | Supported iOS  | Mac Supported |
+| ETC1 | Supported | Not Supported | Supported Android  | Supported Android  | Not Supported |
+| ETC2 | Partially Supported | Not Supported | Not Supported | Supported Android  | Not Supported |
+| ASTC | Partially Supported  | Partially Supported | Not Supported | Partially Supported  | Mac Supported |
+
+By default, **Cocos Creator 3D** outputs the original image during build. If you need to compress an image during the build process, you can select this image in the **Assets Panel** and then manage it in the **Inspector** to edit the compress texture format of the image.
+
+![compress-texture](compress-texture/compress-texture.png)
+
+The editor will provide a preset by default. If you need to add more presets, you can click the `Edit preset` button to open `Project Setting -> Compress Texture` to add and edit presets. The compression format here is readonly. For how to add texture compression presets, please refer to [Project Settings](./editor/project/index.md).
```

Comment:
please refer to [Project Settings](./editor/project/index.md).
-->
please refer to the [Project Settings](./editor/project/index.md) documentation.

Reply:
You missed "the"
--> please refer to the